api/src/services/roles.ts

```diff
@@ -184,6 +184,16 @@ export class RolesService extends ItemsService {
 		}
 	}
 
+	private getRoleAccessType(data: Partial<Item>) {
+		if ('admin_access' in data && data['admin_access'] === true) {
+			return 'admin';
+		} else if ('app_access' in data && (data['app_access'] === true || 'app_access' in data === false)) {
```

Comment:
Seems to me if the first condition `'app_access' in data` is `true` then the `'app_access' in data === false` inside the second condition will always be `false` hence never influencing the `||` and making it redundant

Reply:
Good catch 👍 I guess it's supposed to be

```suggestion
		} else if (('app_access' in data && data['app_access'] === true) || 'app_access' in data === false) {
```